src/sortWith.js

```diff
@@ -0,0 +1,45 @@
+var _curry2 = require('./internal/_curry2');
+
+
+/**
+ * Sorts a list according to a list of comparators.
+ *
+ * @func
+ * @memberOf R
+ * @category Relation
+ * @sig [a -> a -> Number] -> [a] -> [a]
+ * @param {Array} functions A list of comparator functions.
+ * @param {Array} list The list to sort.
+ * @return {Array} A new list sorted according to the comarator functions.
+ * @example
+ *
+ *      var alice = {
+ *        name: 'alice',
+ *        age: 40
+ *      };
+ *      var bob = {
+ *        name: 'bob',
+ *        age: 30
+ *      };
+ *      var clara = {
+ *        name: 'clara',
+ *        age: 40
+ *      };
+ *      var people = [clara, bob, alice];
+ *      var ageNameSort = R.sortWith([
+ *        R.descend(R.prop('age')),
+ *        R.ascend(R.prop('name'))
+ *      ]);
+ *      ageNameSort(people); //=> [alice, clara, bob]
+ */
+module.exports = _curry2(function sortWith(fns, list) {
+  return Array.prototype.slice.call(list, 0).sort(function(a, b) {
```

Comment:
true, but if I'm not wrong then `sortWith` would allow devs to ensure consistent sort order, for example by having the final `comparator` be something like `R.ascend(R.prop('id'))`. This is actually one of the primary use cases that I have for this function, its not just that stable sort isn't guaranteed, some API responses don't have guaranteed sort order either.